services/app-web/src/pages/StateDashboard/StateDashboard.tsx

```diff
@@ -66,7 +66,7 @@ export const StateDashboard = (): React.ReactElement => {
                 getCurrentRevisionFromHealthPlanPackage(sub)
 
             if (currentRevisionDataOrError instanceof Error) {
-                return null
```

Comment:
This should return something. How about the `<ErrorAlertFailedRequest />` UI used in CMS dashboard. Want to avoid returning null from a page level component in general, we should be using fall back error pages.

Reply:
Yeah, this is why I try and avoid using forEach() because I always get confused about where the return will go. This is fine for now but hopefully can be better handeled by another wrapper.